typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -82,30 +84,107 @@ const chainProtocolFee: Record<ChainName, string> = {
   zircuit: '400000000000000',
 };
 
-export function getRenzoHook(
-  defaultHook: Address,
-  chain: ChainName,
-  owner: Address,
-): HookConfig {
+export function getRenzoHook(params: {
+  defaultHook: Address;
+  origin: ChainName;
+  destinationChains: ChainName[];
+  owner: Address;
+  useLegacyRoutingHook: boolean;
```

Comment:
nit: set the default value to true so that you dont have to update call sites in the diff that are not relevant